### PR TITLE
Added old version

### PR DESCRIPTION
**Pull request by seifibrahim32**
## Changelog Description
Changed Publisher  from old version

### CODE REVIEW

**Comment by seifibrahim32** [comment]
@eng-EslamEzzat Check Again

File: openpype/tools/publisher/control.py
Comment (eng-EslamEzzat) [inline openpype/tools/publisher/control.py:1]: why did you delete the code..!!

File: openpype/tools/publisher/window.py
Comment (eng-EslamEzzat) [inline openpype/tools/publisher/window.py:1]: You have to make sure about all lines you added and removed

File: openpype/tools/workfiles/files_widget.py
Comment (eng-EslamEzzat) [inline openpype/tools/workfiles/files_widget.py:1]: why you removed selectionModel and why there is no a replacement for it

**Review by eng-EslamEzzat — CHANGES_REQUESTED** [review]
Please edit files reviewed

File: openpype/tools/publisher/control.py
Comment (seifibrahim32) [reply openpype/tools/publisher/control.py:1]: I return this snippet code back

File: openpype/tools/publisher/window.py
Comment (seifibrahim32) [reply openpype/tools/publisher/window.py:1]: I made it similar and I reverted it back

Comment (eng-EslamEzzat) [reply openpype/tools/publisher/window.py:1]: check it again

File: openpype/tools/workfiles/files_widget.py
Comment (eng-EslamEzzat) [reply openpype/tools/workfiles/files_widget.py:1]: change the "currentChanged"

File: openpype/vendor/python/python_2/PySide-1.2.4.dist-info/RECORD
Comment (eng-EslamEzzat) [inline openpype/vendor/python/python_2/PySide-1.2.4.dist-info/RECORD:1]: make the files similar

**Review by eng-EslamEzzat — CHANGES_REQUESTED** [review]

File: openpype/tools/publisher/publish_report_viewer/widgets.py
Comment (eng-EslamEzzat) [inline openpype/tools/publisher/publish_report_viewer/widgets.py:1]: check the currentChanged

File: openpype/tools/publisher/widgets/list_view_widgets.py
Comment (eng-EslamEzzat) [inline openpype/tools/publisher/widgets/list_view_widgets.py:1]: check the currentChanged

File: openpype/tools/publisher/widgets/overview_widget.py
Comment (eng-EslamEzzat) [inline openpype/tools/publisher/widgets/overview_widget.py:1]: what should the updateCurrentValue return ?

**Review by eng-EslamEzzat — CHANGES_REQUESTED** [review]

File: openpype/tools/publisher/widgets/overview_widget.py
Comment (eng-EslamEzzat) [reply openpype/tools/publisher/widgets/overview_widget.py:1]: try the original first

**Review by eng-EslamEzzat — CHANGES_REQUESTED** [review]

File: openpype/tools/workfiles/files_widget.py
Comment (seifibrahim32) [reply openpype/tools/workfiles/files_widget.py:1]: I fixed it, check back.

File: openpype/tools/publisher/widgets/overview_widget.py
Comment (seifibrahim32) [reply openpype/tools/publisher/widgets/overview_widget.py:1]: I fixed it, check back.

File: openpype/tools/publisher/widgets/list_view_widgets.py
Comment (seifibrahim32) [reply openpype/tools/publisher/widgets/list_view_widgets.py:1]: I fixed it, check back.

File: openpype/tools/publisher/publish_report_viewer/widgets.py
Comment (seifibrahim32) [reply openpype/tools/publisher/publish_report_viewer/widgets.py:1]: I fixed it, check back.

File: openpype/tools/publisher/control.py
Comment (seifibrahim32) [reply openpype/tools/publisher/control.py:1]: I fixed it, check back.

File: openpype/tools/publisher/widgets/overview_widget.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-from qtpy import QtWidgets, QtCore
+from qtpy import QtWidgets, QtCore, QtGui
```
Comment (eng-EslamEzzat) [inline openpype/tools/publisher/widgets/overview_widget.py]: Why you use QtGui?

File: openpype/tools/workfiles/files_widget.py [hunk 2]
```diff
@@ -67,7 +67,8 @@ def __init__(self, parent):
             "Please choose context on the left<br/>&lt",
             self
         )
-        label_widget.setAlignment(QtCore.Qt.AlignCenter)
+        label_widget.setAlignment(QtCore.Qt.
```
Comment (eng-EslamEzzat) [inline openpype/tools/workfiles/files_widget.py]: make it in oneline!

**Review by eng-EslamEzzat — CHANGES_REQUESTED** [review]

File: openpype/tools/publisher/widgets/overview_widget.py (hunk 1 above)
Comment (seifibrahim32) [reply openpype/tools/publisher/widgets/overview_widget.py]: OK, well I will try to use the orignial version so maybe we used the previous version as well

File: openpype/tools/workfiles/files_widget.py (hunk 2 above)
Comment (seifibrahim32) [reply openpype/tools/workfiles/files_widget.py]: ok i do it

**Review by eng-EslamEzzat — APPROVED** [review]
ok, we can merge your commit now!
